src/sentry/web/urls.py

```diff
@@ -68,6 +67,14 @@
         )
     ]
 
+# demo mode
+if settings.DEMO_MODE:
+    from sentry.web.frontend.demo_start import DemoStartView
+
+    urlpatterns += [
+        url(r"^demo/start/", DemoStartView.as_view(), name="sentry-demo-start"),
```

Comment:
You probably want a `$` here since it's not a prefix match.

Reply:
I think longer term, what'll be more sustainable is how we do this for the `debug_urls`, and move all the url routes inside of `sentry.demo.urls` or something. But this is fine for now.